shims/spark311cdh/src/main/scala/org/apache/spark/sql/rapids/shims/spark311cdh/ShuffleManagerShim.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020-2021, NVIDIA CORPORATION.
```

Comment:
remove 2020... leave this file in the shim because we specifically have a version of this in each shim since its a config